docs/topics/compatibility-guides/compatibility-guide-22.md

```diff
@@ -23,6 +23,22 @@ perspective (for example, from Java) is out of the scope of this document.
 
 ## Language
 
+### Remove language versions 1.6 and 1.7
+
+> **Issue**: [KT-71793](https://youtrack.jetbrains.com/issue/KT-71793)
+>
+> **Component**: Core language
+>
+> **Incompatible change type**: source
+>
+> **Short summary**: Kotlin 2.2 introduces language version 2.2 and removes support for language versions 1.6 and 1.7.
+> Language versions 1.8 and 1.9 are deprecated.
```

Comment:
I see that many people are confused by this description, they might think that we’re dropping backward compatibility with the mentioned versions, which is not the case.

Maybe we can phrase it like this:

“Starting from Kotlin compiler 2.2, it’s no longer possible to use -language-version=1.6 or -language-version=1.7, i.e. language feature sets older than 1.8 are no longer supported. However, the language itself remains fully backward compatible with Kotlin 1.0.”

Additionally, I would update documentation for -language-version flag on our page https://kotlinlang.org/docs/compiler-reference.html#language-version-version